### PR TITLE
ex5.1_this_is_the_way

### PR DESCRIPTION
**Pull request by racheldekel**

### CODE REVIEW

File: ex5.1_this_is_the_way.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# function that print the file in folder that start with "deep"
+def print_deep_folder():
+    folder_path = input(INPUT_FOLDER_MESSAGE)
+    directory_list = os.listdir(folder_path)
+    only_deep = [f for f in directory_list if f.startswith("deep")]
+    print(only_deep)
+
+#print the program
+if __name__ == '__main__':
+    print_deep_folder()
```
Comment (talbarfm) [inline ex5.1_this_is_the_way.py]: Add a new line at eof (end of file)

File: ex5.1_this_is_the_way.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
+INPUT_FOLDER_MESSAGE="insert the path to the folder:"
+# function that print the file in folder that start with "deep"
+def print_deep_folder():
```
Comment (talbarfm) [inline ex5.1_this_is_the_way.py:4]: I dont the the folder is deep - pick a better name

Comment (talbarfm) [inline ex5.1_this_is_the_way.py:4]: function documentation should look like this -
Capture

File: ex5.1_this_is_the_way.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    folder_path = input(INPUT_FOLDER_MESSAGE)
+    directory_list = os.listdir(folder_path)
+    only_deep = [f for f in directory_list if f.startswith("deep")]
```
Comment (talbarfm) [inline ex5.1_this_is_the_way.py]: dont use single parameter names

File: ex5.1_this_is_the_way.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import os
+INPUT_FOLDER_MESSAGE="insert the path to the folder:"
+# function that print the file in folder that start with "deep"
+def print_deep_folder():
+    folder_path = input(INPUT_FOLDER_MESSAGE)
```
Comment (talbarfm) [inline ex5.1_this_is_the_way.py]: receive this as a function parameter and access a constant from the outside - it makes you code coupled with a specific path

**Review by talbarfm — CHANGES_REQUESTED** [review]

**Review by talbarfm — COMMENTED** [review]
This ia duplicate pull request - use the comments here and close it

**Review by lovodkin93 — APPROVED** [review]
@racheldekel good job!

**Pull request closed without merging**